node/clients/profile/profileV1.ts

```diff
@@ -15,7 +15,7 @@ function mapAddressesObjToList(addressesObj: any): Address[] {
   )
 }
 
-export class ProfileClient extends JanusClient {
+export class ProfileClientV1 extends JanusClient {
```

Comment:
I needed a way to differentiate between the two and I could only think of versions. If I drop the V1 I'd need to add some prefix because we already have the `ProfileClient` which is a facade for these inner clients. In this case, do you have a better postfix?

Reply:
Nope, sounds good for me.